### DISCUSSION TITLE
Member's blog sites

### DISCUSSION THREAD

**Discussion by fumitoh** [actuarialopensource/website#36] [Team Posts]
In addition to the awesome list, how about making a list of member's individual web sites?
Here are what I could find. Add more!

* @genedan  https://genedan.com/
* @Houstonwp https://houstonp.com/
* @paddyhoran https://www.remotely-actuarial.com
* @wjwillemse https://mangosaurus.eu/
* @franciscogarate http://garpa.net

**Comment by paddyhoran** [comment]
I think it's a good idea.  I was not aware of some of these and there is some interesting reading here.  I forgot mine is still up, I need to update it 🙂.
